src/ui/NoNameFacet/NoNameFacetValues/NoNameFacetValueElement.ts

```diff
@@ -0,0 +1,109 @@
+import { $$ } from '../../../utils/Dom';
+import { NoNameFacet } from '../NoNameFacet';
+import { NoNameFacetValueCheckbox } from './NoNameFacetValueCheckbox';
+import { NoNameFacetValue } from './NoNameFacetValue';
+import { NoNameFacetValueUtils } from './NoNameFacetValueUtils';
+import { l } from '../../../strings/Strings';
+import { KeyboardUtils, KEYBOARD } from '../../../utils/KeyboardUtils';
+
+export class NoNameFacetValueElement {
+  public element: HTMLElement;
+  private labelElement: HTMLElement;
+  private labelWrapperElement: HTMLElement;
+  private captionElement: HTMLElement;
+  private countElement: HTMLElement;
+  private checkboxElement: HTMLElement;
+
+  constructor(private facet: NoNameFacet, private facetValue: NoNameFacetValue) {
```

Comment:
I *think* that this could be somewhat merged with the `NoNameFacetValue` class, which could accept a `Renderer` class instead.

And the `NoNameFacetValue` would simply expose a `render()` method or something similar, which would simply pass down to the Renderer class.

If we need to render in a different way (or if users need more precise control on how to modify the rendering with custom code), it seems like it would be trivial to simply inject a new Renderer class.